index.ts

```diff
@@ -40,6 +40,9 @@ export async function generateOverview(startDate: string) {
           issuesCreated: 0,
           bountiedIssuesCount: 0,
           bountiedIssuesTotal: 0,
+          approvalsGiven: 0,
+          changesRequested: 0,
+          changesRequestedGiven: 0,
         }
```

Comment:
Its hard to know the difference between these, is there a better name for changes requested? The given one i understand, i think rejectionsReceived might be the term youre looking for?

Reply:
Also “changes requested” should really just be called “rejection”, its much more clear